modules/ROOT/pages/migration/index.adoc

```diff
@@ -20,6 +20,152 @@ npm update @neo4j/graphql
 
 Here is a list of all the breaking changes from version 5.0.0 to 6.0.0.
 
+=== Changed minimum Node.js version to 20.0.0
+
+The minimum Node.js version required to run the Neo4j GraphQL Library is now 20.0.0.
+
+=== Changed minimum Neo4j version to 5.0.0
+
+The minimum Neo4j version required to run the Neo4j GraphQL Library is now 5.0.0.
+
+=== Removed the deprecated implicit "some" filter from `@relationship`
+
+The deprecated "some" filter without operator suffix has been removed from `@relationship` in favor of the explicit `_SOME` filter.
```

Comment:
```suggestion
The deprecated `some` filter without operator suffix has been removed from `@relationship` in favor of the explicit `_SOME` filter.
```

I'm not 100% sure, but I think that this makes it more consistent.

Reply:
hm... i used " " because it's implicit. i wanted to use different markup than for keywords which actually exist

Reply:
That's a good point.